### PR TITLE
Lido on Polygon Admin alerts

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: polygon/src/agent-dao-ops.ts [hunk 1]
```diff
@@ -134,3 +147,110 @@ async function handleBufferedMatic(
...
+          alertId: eventInfo.alertId,
+          severity: eventInfo.severity,
+          type: eventInfo.type,
+          metadata: { args: String(event.args) },
+        }))
+      }
+    }
+  }
+  )
+}
```
Comment (vgorkavenko) [inline polygon/src/agent-dao-ops.ts:256]: blank line

File: polygon/src/agent-dao-ops.ts [hunk 2]
```diff
@@ -134,3 +147,110 @@ async function handleBufferedMatic(
...
+  }
+}
+
+export async function handleTransaction(txEvent: TransactionEvent) {
+  const findings: Finding[] = [];
+  await Promise.all([
+    handleProxyAdminEvents(txEvent, findings),
+    handleStMaticTx(txEvent, findings),
+  ]
+  )
```
Comment (vgorkavenko) [inline polygon/src/agent-dao-ops.ts:212]: not nice formatting

File: polygon/src/agent-dao-ops.ts [hunk 3]
```diff
@@ -134,3 +147,110 @@ async function handleBufferedMatic(
...
+          description: eventInfo.description(event.args),
+          alertId: eventInfo.alertId,
+          severity: eventInfo.severity,
+          type: eventInfo.type,
+          metadata: { args: String(event.args) },
+        }))
+      }
+    }
+  }
+  )
```
Comment (vgorkavenko) [inline polygon/src/agent-dao-ops.ts:255]: and this one

File: polygon/src/constants.ts [hunk 4]
```diff
@@ -1,21 +1,116 @@
...
+  stMATIC_proxy: "0x9ee91F9f426fA633d227f7a9b000E28b9dfd8599",
+  validator_factory_proxy: "0x0a6C933495a7BB768d95f4667B074Dd5b95b78eB",
+  node_operator_registry_proxy: "0x797C1369e578172112526dfcD0D5f9182067c928",
+};
+
+// EVENT ABIs
+export const PROXY_ADMIN_OWNERSHIP_TRANSFERRED =
+  "event OwnershipTransferred (address indexed previousOwner, address indexed newOwner)";
+
+export const ST_MATIC_ADMIN_EVENTS = [
```
Comment (vgorkavenko) [inline polygon/src/constants.ts:27]: It seems to make sense to create a type (like `StMaticAdminEvent`) and specify it in the places used (arrow functions and etc.)

Comment (dgusakov) [reply polygon/src/constants.ts:30]: I'm not sure that i got you right. Pls provide some example

Comment (vgorkavenko) [reply polygon/src/constants.ts:30]:
```
type StMaticAdminEvent = {
	address: string
	event: string
	alertId: string
	name: string
	description: CallableFunction
	severity: string // or forta specific type ?
	type: string // or forta specific type ?
}
```

Comment (dgusakov) [reply polygon/src/constants.ts]: Fixed

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**